src/main/resources/site/site.xml

```diff
@@ -0,0 +1,3 @@
+<!-- Delete this file if you are building a web app -->
+<site/>
+</site>
```

Comment:
No. it's either `<site></site>` or `<site/>` (the last one is better).

Reply:
😆 I just commited your code Alan